Formula/lima.rb

```diff
@@ -0,0 +1,29 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima.git",
+    tag:      "v0.3.0",
+    revision: "c82737b29cf54225c3ec8ec7073609d42924017c"
+  license "Apache-2.0"
+  head "https://github.com/AkihiroSuda/lima"
+
+  depends_on "go" => :build
+
+  depends_on "qemu"
+
+  def install
+    system "make", "clean", "binaries"
+
+    bin.install Dir["_output/bin/*"]
+    share.install Dir["_output/share/*"]
+
+    # Install bash completion
+    output = Utils.safe_popen_read("#{bin}/limactl", "completion", "bash")
+    (bash_completion/"limactl").write output
+  end
+
+  test do
+    s_output = shell_output("#{bin}/limactl prune 2>&1")
+    assert_match "Pruning", s_output
```

Comment:
```suggestion
    assert_match "Pruning", shell_output("#{bin}/limactl prune 2>&1")
```
There's no need to store this in a variable.

Reply:
Fine for me